--- a/Makefile	2013-06-23 22:14:45.000000000 -0400
+++ b/Makefile	2015-02-11 23:13:52.878134280 -0500
@@ -26,8 +26,8 @@
 
 all: $(MENU_FILES) $(DESKTOP_FILES) $(DIRECTORY_FILES) $(ICONS)
 install: all
-	install -d "$(DESTDIR)$(EPREFIX)/etc/xdg/applications-merged"
-	install -m0644 $(MENU_FILES) "$(DESTDIR)$(EPREFIX)/etc/xdg/applications-merged"
+	install -d "$(DESTDIR)$(EPREFIX)/etc/xdg/menus/applications-merged"
+	install -m0644 $(MENU_FILES) "$(DESTDIR)$(EPREFIX)/etc/xdg/menus/applications-merged"
 	install -d "$(DESTDIR)$(EPREFIX)/usr/share/applications"
 	install -m0644 $(DESKTOP_FILES) "$(DESTDIR)$(EPREFIX)/usr/share/applications"
 	install -d "$(DESTDIR)$(EPREFIX)/usr/share/desktop-directories"
